### ISSUE TITLE
How to make prediction with custom dataset (Inception)

### ISSUE THREAD

**Issue by kushia** [tensorflow/models#584]
Hi,
I tried to create a custom dataset of image from scratch with the inception model.
So I created my TFreaders, did the training (With imagenet_train with bazel-bin).
At this steps, I had one ckpt file and one meta file.
But I don't know what to do with those files to make predictions now ?

I tried to print the graph and freeze it according to my search on stackoverflow (With freeze_graph.py), but I don't know to what to put on output_node_name.. (I tried save/restore_all, final_result, tower_0/logits/predictions) but nothing seems to be work.

So I'm actually kind of lost..
Would you have some ideas ?
I don't want to use the retrain script cause I want the network to be specialized on my dataset..
Thanks everyone!

**Comment by drpngx** [comment]
This is probably best asked on stackoverflow. Could you post there with the tag "tensorflow" (we monitor the issues).

**Comment by kushia** [comment]
Thanks to you.
Posted on github, in the wait of your answers.

**Comment by drpngx** [comment]
You mean stackoverflow, right?

**Comment by kushia** [comment]
Just in case, the link of the question on stackoverflow :
http://stackoverflow.com/questions/40247104/tensorflow-inception-cant-make-predictions

2016-10-26 0:04 GMT+02:00 Jean Yim <yim.jean@gmail.com>:

> Yes sorry, I was distracted..
> Thanks you !
>

**Comment by drpngx** [comment]
Nice, thanks!

**Comment by kushia** [comment]
Yes sorry, I was distracted..
Thanks you !
